src/options/options.html

```diff
@@ -1,214 +1,214 @@
 <!DOCTYPE html>
 <html lang="en">
-	<head>
-		<meta charset="utf-8">
-		<meta name="viewport" content="width=device-width, initial-scale=1.0">
-		<link rel="stylesheet" href="../common/common.css">
-		<link rel="stylesheet" href="options.css">
+  <head>
```

Comment:
Please do not change indentation and use `editorconfig`, which [requires tabs for HTML](https://github.com/rugk/offline-qr-code/blob/master/.editorconfig#L21).